addon-test-support/fire-event.js

```diff
@@ -31,10 +32,14 @@ export function fireEvent(element, type, options = {}) {
       clientY: y
     };
     event = buildMouseEvent(type, merge(simulatedCoordinates, options));
+  } else if (FILE_SELECTION_EVENT_TYPES.indexOf(type) > -1 && element.files) {
```

Comment:
Ok, I can buy the idea. Triggering a change event on an file input is a resonable condition to check. 
However, will this work if the element is not a multiple file input? Or in that case it would be `file`?